test/xunit.analyzers.tests/CodeAnalyzerHelper.cs

```diff
@@ -15,6 +15,7 @@ class CodeAnalyzerHelper
     {
         static readonly MetadataReference CorlibReference = MetadataReference.CreateFromFile(typeof(object).GetTypeInfo().Assembly.Location);
         static readonly MetadataReference SystemCoreReference = MetadataReference.CreateFromFile(typeof(Enumerable).GetTypeInfo().Assembly.Location);
+        static readonly MetadataReference SystemTextReference = MetadataReference.CreateFromFile(typeof(System.Text.RegularExpressions.Regex).GetTypeInfo().Assembly.Location);
```

Comment:
I think this is actually a reference to System.dll (at least on desktop), which doesn't look like it was explicitly referenced here yet.